doc/source/working_with_kiwi.rst

```diff
@@ -84,3 +85,149 @@ to find in the *description directory*:
           <archive name="custom-archive.tgz"/>
       </packages>
 
+
+.. _working-with-kiwi-image-building-process:
+
+Image Building Process
+----------------------
+
+KIWI creates images in a two step process. The first step, the *prepare*
+operation, generates a so-called *unpacked image* tree (directory) using
```

Comment:
Maybe use "unpacked image tree" as a (new) term. Therefor, I would add "tree" into the italic text:

```suggestion
operation, generates a so-called *unpacked image tree* (directory) using
```